docs/PIPELINES-CONFIG.md

```diff
@@ -0,0 +1,275 @@
+    ```sh
+    kubectl get configmap backstage-dynamic-plugins-ai-rh-developer-hub -n ai-rhdh -o yaml | yq '.data["dynamic-plugins.yaml"]' > temp-dynamic-plugins.yaml
+    ```
+2. Merge the contents of [`tekton-plugins.yaml`](../dynamic-plugins/tekton-plugins.yaml) into the temp file
+    ```sh
+    yq -i ".plugins += $(yq '.plugins' ./dynamic-plugins/tekton-plugins.yaml -M -o json) | .plugins |= unique_by(.package)" temp-dynamic-plugins.yaml
+    ```
+3. Patch the dynamic plugins ConfigMap with the updated content in the temp file
+    ```sh
+    kubectl patch configmap backstage-dynamic-plugins-ai-rh-developer-hub -n ai-rhdh \
+    --type='merge' \
+    -p="{\"data\":{\"dynamic-plugins.yaml\":\"$(echo "$(cat temp-dynamic-plugins.yaml)" | sed 's/"/\\"/g' | sed 's/$/\\n/g' | tr -d '\n')\"}}"
+    ```
+4. Dynamic plugins should be updated with the [Tekton plugins list](../dynamic-plugins/tekton-plugins.yaml) with a pod update triggered and you may remove the temp file at this point
+
+#### Step 9: Updating RHDH Deployment
+
+We need to map the referenced environment variable `K8S_SA_TOKEN` to the `rhdh-kubernetes-plugin` service account token secret.
+
+We can get the name of the service account tied token secret with the following command:
+
+```sh
+kubectl get secrets -n "<rhdh_namespace>" -o name | grep rhdh-kubernetes-plugin-token- | cut -d/ -f2 | head -1
+```
+
+Using the returned name from the command above, we can bind this secret to the `K8S_SA_TOKEN` under `.spec.template.spec.containers.env`
+
+![Tekton Secret Addition](../assets/tekton-secret-example.png)
+
+### Pre-Existing Instance: Script Configuration
+
+#### Step 1: Required Information
+
+If you have your own RHDH instance created you can configure it to work with the pipelines service with the use of our config script. You will need the following information on hand:
+
+1. Namespace of RHDH instance
+2. Deployment name of RHDH
+3. ConfigMap name for your RHDH plugins
+   1. Typically `dynamic-plugins` for `Helm` installs and `backstage-dynamic-plugins-<name of developer hub instance>` for `Operator` installs
+4. Kubernetes API service account
+
+#### Step 2: Environment Variables
+
+You are able to store these values in environment variables or paste them during the interactive prompts at runtime. Set the following the use environment variables:
+- `$EXISTING_NAMESPACE`
+  - Name of target RHDH namespace
+- `$EXISTING_DEPLOYMENT`
+  - Name of target RHDH deployment
+- `$RHDH_PLUGINS`
+  - Name of the dynamic plugins ConfigMap
+
+#### Step 3: Configure cosign
+
+The installer will configure and set up cosign that the `configure-pipeline.sh` script needs to reference the cosign public key to set to the cosign secret under the deployment namespaces.
+
+To configure cosign to sign secrets follow these steps:
+1. Download `cosign` cli tool
+    ```sh
+    curl -L https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64 -o cosign && chmod +x cosign
```

Comment:
I receive an exec format error on the binary, we may need to have this dynamically set depending which architecture the script is being run on?